04/cipher_caesar.py

```diff
@@ -0,0 +1,41 @@
+diacritics = ['á', 'é', 'í', 'ó', 'ú', 'ý', 'č', 'ď', 'ě', 'ň', 'ř', 'š', 'ť', 'ž', 'ů']
+ASCII_numbers = [49, 50, 51, 52, 53, 54, 55, 56, 57, 58]
+
+key = input('key ')
+for znak in key:
+    znak_ASCII = ord(znak)
+    if znak_ASCII not in ASCII_numbers:
+        key = int(input('Zadávej jen celá, kladná čísla. Příklad: 2, 5, 15 \nKey: '))
+
+key = int(key)
+
+if key <= 0:
+    key = int(input('Key můžou být jen celá, kladná čísla. Příklad: 2, 5, 15 \nKey: '))
+
+plain_text = input('plain text ')
+for character in plain_text:
+    if character in diacritics:
+        plain_text = input('Bez čísel a diakritiky. Příklad: les, pes, jedna \nPlain text: ')
+    elif character.isnumeric():
+        plain_text = input('Bez čísel a diakritiky. Příklad: les, pes, jedna \nPlain text: ')
+plain_text = plain_text.lower()
```

Comment:
Bolo by fajn keby si zachovala veľké/malé písmená. Skús sa nad tým zamyslieť ako by si to mohla urobiť. Napr. ak by si šifrovala slovo `Python` kľúčom 2, tak by výsledok mal byť `Ravjqp`.

Reply:
OK, přidám tam rozsah v ASCII